lottie/src/main/java/com/airbnb/lottie/parser/moshi/JsonReader.java

```diff
@@ -188,7 +188,7 @@
   static {
     REPLACEMENT_CHARS = new String[128];
     for (int i = 0; i <= 0x1f; i++) {
-      REPLACEMENT_CHARS[i] = String.format("\\u%04x", (int) i);
+      REPLACEMENT_CHARS[i] = String.format("\\u%04x", i);
```

Comment:
Please don't format any code in the moshi package. It was copied from moshi directly and changing it will make diffing it against future versions harder